velox/docs/functions/spark/math.rst

```diff
@@ -0,0 +1,64 @@
+====================================
+Mathematical Functions
+====================================
+
+.. js:function:: abs(x) -> [same as x]
+
+    Returns the absolute value of ``x``.
+
+.. function:: add(x, y) -> [same as x]
+
+    Returns the result of adding x to y. The types of x and y must be the same.
+    For integral types, overflow results in an error. Function add corresponds 
+    to js's operator ``+``.
+
+.. js:function:: ceil(x) -> [same as x]
+
+    Returns ``x`` rounded up to the nearest integer.  
+    In js only long, double, and decimal have ceil.
+
+.. function:: divide(x, y) -> [same as x]
+
+    Returns the results of dividing x by y. The types of x and y must be the same.
+    It always performs floating point division. Function divide corresponds to js's operator ``/``.
```

Comment:
What is "js's"?

Reply:
Sorry, It's my mistake.  Changed all js's to spark's.